packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue

```diff
@@ -288,6 +294,28 @@ const chartProxy = chartActionsProxy(props.node, (state: SimulateCiemssOperation
 	emit('update-state', state);
 });
 
+const preparedCharts = computed(() => {
```

Comment:
Probably not, because it needs an a workflow-node instance (to get width, height and other attributes). This makes the code a bit awkward, because then the operator will need to know about the node, while the node need to know about the operator ... kind of a circular dependency.

More appropriate would be workflow/util.ts - where we host the common things mostly at the workflow node/drilldown level. I would like to get a few more charts done before doing this (e.g a 2nd pass at interventions), to see if there is an overarching emergent pattern.